File: holamundo.go
package main

import (
	"fmt"
	"log"
	"net/http"
)

func main() {
	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		fmt.Fprintf(w, "Hola Mundo GO")
	})
	fmt.Println("Server listening in port 8080 ...")
	log.Fatal(http.ListenAndServe(":8080", nil))

}

File: holamundo.js
const http = require('http');

const requestListener = function (req, res) {
  res.writeHead(200);
  res.end('Hola Mundo Nodejs');
}

const server = http.createServer(requestListener);
console.log("Server started in port 8080 ...")
server.listen(8080);